src/api/middleware/controllers/review.js

```diff
@@ -0,0 +1,11 @@
+const Review = require("../../../models/Review");
+const { isValidObjectId } = require("../../../utils.js/validator");
+
+module.exports = {
+  create: async (req, res) => {
+    const { reviewText, stars, likes, user } = req.body;
+
+    if (!isValidObjectId(user))
+      return res.json({ message: "user is invalid " });
+  },
+};
```

Comment:
This implementation seems incomplete, is it deliberate maybe to be completed later 🤔?